lesson_5/lesson_5.py

```diff
@@ -1,6 +1,47 @@
 # coded: UTF-8
 
-# 1 Task Version 2
+# 1 Task Version 3 (Match\Case with RegEx and Exception)
```

Comment:
This version is my practice, i add this version because i didn't want to lose it.